automation_use_cases/scan_automation/bin/main.py

```diff
@@ -0,0 +1,37 @@
+import sys
+sys.path.append("..")
+import os
+import getpass
+import logging
+from ruamel.yaml import YAML
+from datetime import datetime
+from lib.helpers import scan_automation
+
+
+def main():
+    # Load configuration file
+    yaml = YAML(typ='safe')
+    settings = yaml.load(open("../config/settings.yml"))
+    
+    # Setup basic logging
+    logging.basicConfig(filename="../log/scan_automation.log", filemode="a", level=logging.INFO,
+                        format=str(datetime.now()) + " - %(levelname)s - %(message)s")
+
+    # Get connection info
+    if settings.get("connection"):
+        api_key = settings.get("connection").get("api_key")
+        region = settings.get("connection").get("region", "us")
+    else:
+        api_key = None
+        region = "us"
+
+    # Prompt user for API key if there isn't one configured
+    if api_key is None:
+        api_key = getpass.getpass("Please enter your InsightAppSec API key:")
+
```

Comment:
Thoughts of using `getopt` so a user could not define an app/scan config in the configuration file and simply pass it in as command line arguments?  Something like: `python main.py --app app1 --scan_config config1`.  If sent in as an argument it would only run what is passed in as arguments and no other configs?